sway/tree/view.c

```diff
@@ -405,14 +404,11 @@ void view_execute_criteria(struct sway_view *view) {
 				criteria->raw, view, criteria->cmdlist);
 		seat_set_focus_container(seat, view->container);
 		list_add(view->executed_criteria, criteria);
-		struct cmd_results *res = execute_command(criteria->cmdlist, NULL);
-		if (res->status != CMD_SUCCESS) {
-			wlr_log(WLR_ERROR, "Command '%s' failed: %s", res->input, res->error);
-		}
+		struct cmd_results *res = execute_command(
```

Comment:
Maybe `seat_set_focus_container` on line 405 can be removed now?

Reply:
Ah, I missed that. Removed now.